code/game/objects/machinery/doors/multi_tile.dm

```diff
@@ -142,7 +142,7 @@
 
 /obj/machinery/door/airlock/multi_tile/mainship/secdoor
 	name = "\improper Security Airlock"
-	icon = 'icons/obj/doors/mainship/2x1secdoor.dmi'
+	icon = 'icons/obj/doors/mainship/2x1secdoor_glass.dmi'
```

Comment:
In the port, CM's 2x1secdoor_glass.dmi is 95% the same as TGMC's 2x1secdoor.dmi **BUT** CM's 2x1secdoor.dmi is very different. I had to change this and pray no mistakes in porting were made.

Reply:
![image](https://user-images.githubusercontent.com/6610922/157816112-3d1bf8c6-71bb-4545-9f47-6304fd33bb75.png)

Only these maps are affected.